src/components/KPop/KPop.vue

```diff
@@ -459,7 +466,7 @@ export default defineComponent({
   padding: var(--kui-space-80, $kui-space-80) var(--kui-space-60, $kui-space-60);
   text-align: left;
   white-space: normal;
-  z-index: 1000;
+  z-index: v-bind(zIndex);
```

Comment:
```suggestion
  z-index: v-bind('zIndex');
```

Same here and other occurrences